superduperdb/base/datalayer.py

```diff
@@ -987,6 +1009,14 @@ def select_nearest(
         outputs: t.Optional[Document] = None,
         n: int = 100,
     ) -> t.Tuple[t.List[str], t.List[float]]:
+        '''
+        Performs a vector search query on given vector index
```

Comment:
Please implement empty lines between body of docs and the parameters.

Reply:
added to all docstings in components